plugins/foreman_discovery/9.1/index.md

```diff
@@ -0,0 +1,1353 @@
+disks). The registered nodes show up on Discovered Hosts page and provisioning
+can be initiated either manually (via UI/CLI or API) or automatically via
+predefined Discovery Rules.
+
+All communication can be optionally passed through Smart Proxy which has
+direct access both to the provisioning network and Foreman instance:
+
+![Discovery Communication Workflows](/static/images/diagrams/discovery_comm.png)
+
+## 1.1 Release Notes
+
+For each Foreman version there is particular version of Foreman Discovery
+plugin:
+
+<table class="table table-bordered table-condensed">
+  <tr>
+    <th>Foreman version</th>
+    <th>Plugin version</th>
+    <th>Proxy version</th>
+    <th>Image version</th>
+    <th>CLI version</th>
+  </tr>
+  <tr>
+    <td>&lt;= 1.2</td>
+    <td>1.0.2</td>
+    <td>N/A</td>
+    <td>N/A</td>
+    <td>N/A</td>
+  </tr>
+  <tr>
+    <td>= 1.3</td>
+    <td>1.1.0</td>
+    <td>N/A</td>
+    <td>0.1.1</td>
+    <td>N/A</td>
+  </tr>
+  <tr>
+    <td>= 1.4</td>
+    <td>1.2.0</td>
+    <td>N/A</td>
+    <td>0.3.0</td>
+    <td>N/A</td>
+  </tr>
+  <tr>
+    <td>= 1.5</td>
+    <td>1.3.0</td>
+    <td>N/A</td>
+    <td>0.5.0</td>
+    <td>N/A</td>
+  </tr>
+  <tr>
+    <td>= 1.6</td>
+    <td>1.4.0</td>
+    <td>N/A</td>
+    <td>0.6 or 2.0</td>
+    <td>N/A</td>
+  </tr>
+  <tr>
+    <td>= 1.7</td>
+    <td>2.0.0</td>
+    <td>0.1</td>
+    <td>2.0</td>
+    <td>0.0.1</td>
+  </tr>
+  <tr>
+    <td>= 1.8</td>
+    <td>3.0.0</td>
+    <td>1.0.2</td>
+    <td>2.1</td>
+    <td>0.0.1</td>
+  </tr>
+  <tr>
+    <td>= 1.9</td>
+    <td>4.0.0</td>
+    <td>1.0.2</td>
+    <td>2.1</td>
+    <td>0.0.2</td>
+  </tr>
+  <tr>
+    <td>= 1.10</td>
+    <td>4.1.1</td>
+    <td>1.0.3</td>
+    <td>3.0</td>
+    <td>0.0.2</td>
+  </tr>
+  <tr>
+    <td>= 1.11</td>
+    <td>5.0.2</td>
+    <td>1.0.3</td>
+    <td>3.1</td>
+    <td>0.0.2</td>
+  </tr>
+  <tr>
+    <td>= 1.12</td>
+    <td>6.0.0</td>
+    <td>1.0.3</td>
+    <td>3.1</td>
+    <td>0.0.3</td>
+  </tr>
+  <tr>
+    <td>= 1.13</td>
+    <td>7.0.0</td>
+    <td>1.0.3</td>
+    <td>3.2-3.3</td>
+    <td>0.0.3</td>
+  </tr>
+  <tr>
+    <td>= 1.14</td>
+    <td>8.0.0</td>
+    <td>1.0.3</td>
+    <td>3.2-3.3</td>
+    <td>0.0.3</td>
+  </tr>
+  <tr>
+    <td>= 1.15</td>
+    <td>{{page.pluginver}}</td>
+    <td>{{page.proxyver}}</td>
+    <td>{{page.imgver}}</td>
+    <td>{{page.cliver}}</td>
+  </tr>
+</table>
+
+### 1.1.1 Foreman Discovery plugin
+
+**9.1**: Release notes
+
+Mostly bugfix release with new support for notifications, possibility to add
+arbitrary kexec kernel options, taxonomy for discovery rules and some UX
+improvements. The next major release will introduce many improvements in UX.
+
+We have released versions 9.0.0 and 9.1.0 but these were problematic with many
+bugs around manual discovery provisioning and Katello integration. All
+problems were finally solved in version 9.1.1. Make sure you upgrade directly
+to 9.1.1 when preparing production version bump.
```

Comment:
"...when updating production versions." sounds slightly better, IMHO, but I'm not a native speaker.